### PR TITLE
 refactor(auth-server): use returned value from .deviceDelete in devices-and-sessions

### LINKED ISSUE #458: Use the value returned from .deviceDelete in devices-and-sessions
See https://github.com/mozilla/fxa-auth-server/pull/2937#discussion_r266709730 for details

### PR DESCRIPTION
**Pull request by mirefly**
Fixes #458

### CODE REVIEW

**Comment by mirefly** [comment]
Thanks  @vbudhram 
I pushed the update.

File: packages/fxa-auth-server/test/local/routes/devices-and-sessions.js [hunk 1]
```diff
@@ -973,7 +973,6 @@ describe('/account/device/destroy', () => {
       return runTest(route, mockRequest, () => {
         assert.equal(mockDB.deleteDevice.callCount, 1);
         assert.isFalse(mockLog.error.calledOnceWith('deviceDestroy.revokeRefreshToken.error'));
-        assert.isTrue(mockOAuthDb.revokeRefreshTokenById.calledOnceWith(refreshTokenId));
```
Comment (vbudhram) [inline packages/fxa-auth-server/test/local/routes/devices-and-sessions.js]: Sorry I didn't notice this earlier, but we should be back these assertions. You can mock the response by the `db.deleteDevice` method by adding this before each test runs.

```
mockDb.deleteDevice = sinon.spy(async () => refreshTokenId);
```

Comment (mirefly) [reply packages/fxa-auth-server/test/local/routes/devices-and-sessions.js]: Thanks @vbudhram  
I updated the test.

**Comment by mirefly** [comment]
Thanks  @vbudhram

**Pull request merged**